packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -645,6 +646,8 @@ export interface ChatMessage extends MessageCommon {
     // (undocumented)
     messageType: 'chat';
     // (undocumented)
+    metadata?: Record<string, string>;
```

Comment:
are we expecting to expose these in stable potentially as well?

Reply:
Api.md files currently don't respect conditional compilation. This will not be exposed in the stable builds.
Please correct me if am wrong @prprabhu-ms